.github/workflows/release.yml

```
@@ -1,4 +1,4 @@
-name: Manual NPM Publish
+name: Release New Version
```

Easier to read in the UI and clear enough (there aren't multiple release pipelines in here to differentiate against).

```suggestion
name: Release
```